src/network/net_addons.cc

```diff
@@ -93,7 +97,8 @@ static inline std::string get_addons_server_name() {
 }
 
 constexpr unsigned kCurrentListVersion = 3;
-std::vector<AddOnInfo> NetAddons::refresh_remotes() {
+constexpr uint32_t kRefreshPeriod = 600000;  // Only refresh every 10min
```

Comment:
```suggestion
constexpr uint32_t kRefreshPeriod = 10 * 60 * 1000;  // Only refresh every 10min
```
Readability. Also add some blank lines please.